pkgs/development/python-modules/tsfresh/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, pytestCheckHook
+, pytestcov
+, pytest_xdist
+, requests
+, numpy
+, pandas
+, scipy
+, statsmodels
+, patsy
+, scikitlearn
+, tqdm
+, dask
+, distributed
+}:
+
+buildPythonPackage rec {
+  pname = "tsfresh";
+  version = "0.17.0";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "f3db06063116c3cca6ef1ad30dcd50a0cc4a4d0688cc34cc8843d9bd400da969";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+    pytestcov
+    pytest_xdist
+  ];
+
+  # Tries to access the network
+  pytestFlagsArray = [
```

Comment:
Is it possible to use `disabledTests` instead?

Reply:
for an entire directory? no

Reply:
In the next staging merge it will be.